src/pages/Pool/index.tsx

```diff
@@ -165,7 +165,7 @@ function WrongNetworkCard() {
   const theme = useTheme()
 
   return (
-    <div style={{ height: '100vh' }}>
+    <>
```

Comment:
Empty `div` was also causing issues with paddings - now, `Pool` renders similar regardless whether it's invalid network or not.

<img width="421" alt="Screenshot 2022-12-15 at 14 22 44" src="https://user-images.githubusercontent.com/2464966/207953303-cd9d46b4-0fcb-4adb-8807-d66cbf793adc.png">

Reply:
Previously, padding was a bit bigger on the sides, but this keeps it aligned with other pages:

<img width="444" alt="Screenshot 2022-12-15 at 14 05 06" src="https://user-images.githubusercontent.com/2464966/207953378-dba1e3bc-d256-4319-9d94-75f1a46734ed.png">

Other pages:

<img width="402" alt="Screenshot 2022-12-15 at 14 49 30" src="https://user-images.githubusercontent.com/2464966/207953469-0fbae1de-2df2-40ee-9162-a7da800b700c.png">

I still think we should unify horizontal paddings because they're slightly off between Pool/Swap/Tokens, but I would leave it for another PR.